taskmanager/server.go

```diff
@@ -0,0 +1,155 @@
+package taskmanager
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/tcsc/levity/api"
+	"github.com/tcsc/levity/registry"
+	"github.com/tcsc/levity/task"
+)
+
+// Error is an error code describing things that can go wrong with the
+// TaskManager API service.
+type Error int
```

Comment:
If you want to define a custom error, in Go you'd normally use a struct, like:

```go
type NoSuchTaskError struct {
  Message string
}

func (e *NoSuchTaskError) Error() string {
  retur e.Message
}
```

Reply:
Replaced integer code with something similar.